.travis.yml

```diff
@@ -1,6 +1,11 @@
 language: python
 python:
   - "2.7"
+env:
+  - TOXENV=django18
+  # 1.9 and 1.10 known to fail currently
+  #- TOXENV=django19
```

Comment:
Think it's worth doing this for 1.10 as well then? They both fail in different ways.